doc/source/how-to/documenting.rst

```diff
@@ -469,349 +470,210 @@ branch within this repository. This is done by uploading the generated
+For future releases of your project, since the new ``release/X.Y`` branches are
+created from ``main`` with the new configuration, no additional changes are
+required.
 
-Paste the value of the token in the ``Settings/Secrets/Actions`` path under a
-new secret named ``DEPLOY_DOCS_PAT`` in the repository of the project. Use this
-secret in your CI/CD:
-
-.. tab-set::
-
-    .. tab-item:: Using ``tox``
-
-        .. code-block:: yaml
-
-            - name: Deploy
-              if: github.event_name == 'push' && contains(github.ref, 'refs/tags')
-              uses: JamesIves/github-pages-deploy-action@v4.3.0
-              with:
-                token: ${{ secrets.DEPLOY_DOCS_PAT }}
-                repository-name: pyansys/repository-name
-                branch: gh-pages
-                folder: .tox/doc_out
-                clean: true
-
-    .. tab-item:: Without using ``tox``
-
-        .. code-block:: yaml
-
-            - name: Deploy
-              if: github.event_name == 'push' && contains(github.ref, 'refs/tags')
-              uses: JamesIves/github-pages-deploy-action@v4.3.0
-              with:
-                token: ${{ secrets.DEPLOY_DOCS_PAT }}
-                repository-name: pyansys/repository-name
-                branch: gh-pages
-                folder: doc/_build/html
-                clean: true
-
+.. note::
 
+    If you require support for migrating to the multi-version documentation,
+    please contact the ``pyansys-core team`` by writing an e-mail to
+    ``support@pyansys.com``.
 
-Deploy when tagging
-+++++++++++++++++++
 
-Depending on your preferences, you can choose to update the documentation on
-tags only (as done earlier) or on each push. If you want to have your
-documentation deployed on each push to ``main``, change the preceding conditional
-to:
+Access online documentation
+---------------------------
+Documentation for the latest stable release of a PyAnsys library is accessible
+from its repository. The canonical name for the documentation of the project is
+constructed using the following structure:
 
-.. code-block:: yaml
+``https://<product>.docs.pyansys.com``
 
-    if: github.ref == 'refs/heads/main'
+You can generally access the latest development version of the documentation by
+adding the prefix ``dev.`` to the URL for the latest stable release.
 
+.. warning::
 
-Access online documentation
----------------------------
-Documentation for the latest stable release of a PyAnsys library is accessible
-from its repository. You can generally access the latest development version of
-the documentation tracking the ``main`` branch by adding the prefix ``dev.`` to
-the URL for the latest stable release.
+    PyAnsys projects support now multi-version documentation, meaning that
+    stable and development versions are collected under the same webpage. A
+    drop-down button for selecting desired version should be available in the
+    top right corner of the navigation bar in the documentation page.
```

Comment:
```suggestion
    PyAnsys projects support multi-version documentation, meaning that
    stable and development versions are collected under the same webpage. A
    drop-down button for selecting the desired version should be available in the
    top right corner of the navigation bar on the documentation page.
```

Reply:
Google style guide says documentation should be timeless, so no "now" or "in the past" or "in the future" stuff.